coin-flip/script.js

```diff
@@ -2,12 +2,13 @@ function getRandNumber(max) {
 	return Math.floor(Math.random() * max);
 }
 
-var number = getRandNumber(2);
 
-var headsOrTails = "None";
+const flipCoin = () => {
+	const number = getRandNumber(2);
 
-if (number == 1) {
-	document.getElementById("coin").innerHTML = "<h1>You flipped Heads!</h1>";
-} else {
-	document.getElementById("coin").innerHTML = "<h1>You flipped Tails!</h1>";
+	const headsOrTails = number === 1 ? "Heads" : "Tails";
```

Comment:
Not sure if you've seen this before, but this is a [ternary operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Conditional_Operator) or in-line `if`/`else`. You could write the same code like this:
```javascript
let headsOrTails = "";
if(number === 1) {
	headsOrTails = "Heads";
} else {
	headsOrTails = "Tails"
}
```

Oh, and that is indeed _three_ equals signs. Without getting into details too much, three equals signs will likely do what you want when comparing numbers/Booleans/etc. Let me know if you want more info, but the shortest answer as to why this happens is "type coercion".